README.md

```diff
@@ -50,15 +50,15 @@ To update the existing Studio deployment, run the following commands
 
 ```bash
 $ helm dependency update
-$ helm upgrade --install --atomic studio studio/ -n <namespace> -f override.yaml
+$ helm upgrade --install --atomic studio iterative/studio -f override.yaml
```

Comment:
```suggestion
$ helm upgrade --install --atomic studio iterative/studio --namespace <namespace> -f override.yaml
```

I would not drop it. because I would allow the User to decide where he wants to deploy it. 

The change I would set is to clearly name na parameter `--namespace`